shared/domains/chelonia/time-sync.js

```diff
@@ -112,10 +112,10 @@ export default (sbp('sbp/selectors/register', {
   // Get an estimate of the server's current time based on the time elapsed as
   // measured locally (using a monotonic clock), which is used as an offset, and
   // a previously retrieved server time. The time value is returned as a UNIX
-  // timestamp (seconds since 1 Jan 1970 00:00:00 UTC)
+  // _millisecond_ timestamp (milliseconds since 1 Jan 1970 00:00:00 UTC)
   'chelonia/time': function (): number {
     const monotonicNow = performance.now()
     const wallNow = wallBase - monotonicBase + monotonicNow
-    return (wallNow / 1e3 | 0)
+    return Math.round(wallNow)
```

Comment:
Why is `Math.round` being called here?

Reply:
Because this value should be an integer, the extra precision isn't significant. Also, the shelter auth header uses dots as a special character.

Reply:
Ah I see. Well, in that case, it might be the case that `chelonia/time` is used in situations where the decimal place is useful, so could you please move this `.round` to the call-site that calls this selector for generating the header?

Reply:
The decimal place is never useful because it's noise.

Reply:
So what you're saying is that you are personally guaranteeing me that no Chelonia application, whether made by us or anyone else in the future, will ever have have a need for sub-second timestamps? Can you give me some sort of proof of this?

Reply:
Whether they do or don't is irrelevant. JavaScript doesn't offer sub-millisecond timestamps. Note sub-**milli**second.

Reply:
Oh............................................................................. 🤦‍♂️ 

I'm having a day.

Pretend you didn't see any of that.